src/worker.ts

```diff
@@ -132,22 +186,13 @@ async function onCallbackQuery(callbackQuery: CallbackQueryType) {
     // remove tag from issue body
     const tagFreeTitle = removeTag(replyToMessage);
 
-    const { data, assignees } = await createIssue(
-      taskInfo?.timeEstimate || "",
-      orgName,
-      repoName,
-      taskInfo?.title || "",
-      tagFreeTitle,
-      messageLink,
-      tagged || ""
-    );
+    const { data, assignees, error } = await createIssue(timeEstimate || "", orgName, repoName, title || "", tagFreeTitle, messageLink, tagged || "");
 
-    console.log(`Issue created: ${data.html_url}`);
+    console.log(`Issue created: ${data.html_url} ${data.message}`);
 
-    const msg = escapeMarkdown(
-      `*Issue created: [Check it out here](${data.html_url})* with time estimate *${taskInfo?.timeEstimate}*${assignees ? ` and @${tagged} as assignee` : ""}`,
-      "*`[]()"
-    );
+    const msg = data.html_url
+      ? `*Issue created: [Check it out here](${data.html_url})* with time estimate *${timeEstimate}*${assignees ? ` and @${tagged} as assignee` : ""}`
```

Comment:
I also believe I wrote somewhere an adjusted message without the "check it out here" part but I have to look for that and it's a bit slow from my phone.